src/ui/public/management/index.js

```diff
@@ -0,0 +1,23 @@
+import ManagementSection from './section';
+
+const sections = new ManagementSection('management', {
+  display: 'Management'
+});
+
+// TODO: where should this live?
+sections.register('data', {
+  display: 'Connect Data',
+  order: 0
+});
+
+sections.register('elasticsearch', {
+  display: 'Elasticsearch',
+  order: 10
+});
+
+sections.register('kibana', {
+  display: 'Kibana',
+  order: 20,
+});
```

Comment:
> I think it makes sense for the management app to specifically control which categories can exist.

I don't agree with that in the long term, but I do agree that we don't need to solve it in this PR.